sdk/servicebus/azure-servicebus/samples/sync_samples/send_and_receive_amqp_annotated_message.py

```diff
@@ -34,7 +34,7 @@ def send_data_message(sender):
 def send_sequence_message(sender):
     sequence_body = [b'message', 123.456, True]
     footer = {'footer_key': 'footer_value'}
-    properties = {"subject": "sequence"}
+    properties = AmqpMessageProperties(subject="sequence")
```

Comment:
MyPy wants these to be the designated type given in AMQPAnnotatedMessage --- do we also support dict[str,any] or is this a sample error?

Reply:
I think the type of `properties` in the `AmqpAnnotatedMessage` init needs to be updated to accept a dict. Looking at it if properties are sent in to the init, it unpacks it like a dict.